packages/core/src/services/commits/computeRevertChanges.ts

```
@@ -191,9 +191,5 @@ export async function computeChangesToRevertCommit(
     targetDraftDocuments: documentsInDraft.value,
   })
 
-  if (changesToRevert.length === 0) {
-    return Result.error(new ConflictError('No changes to revert'))
```

I think here you can throw the error and will arrive to `onError` Can you check?

---

It does arrive to the `onError` handler, yes, but the sentry notification would already have been sent